makefile.shade

```diff
@@ -203,6 +204,40 @@ var buildTarget = "compile"
         }
     }
 
+#cache-coherence
+    @{
+        var coherenceCacheDir = Path.Combine(BASE_DIR, Environment.GetEnvironmentVariable("COHERENCE_CACHE_DIR") ?? ".coherence-cache");
```

Comment:
I think the build dir should not be relative to the `BASE_DIR` if the `COHERENCE_CACHE_DIR` is set. On the CI we'll probably set it to a global location so that builds share it and they don't have to redownload it.